automatic/opera/opera.nuspec

```diff
@@ -15,11 +15,12 @@
 
 ## Parameters
 
+- `/NoAutostart` - Do not add Opera entry to `HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\Run` registry
 - `/NoDesktopShortcut` - Do not create desktop shortcut for Opera
 - `/NoTaskbarShortcut` - Do not pin Opera to taskbar
 
 These parameters can be passed to the installer with the use of `--params`.
-For example: `--params '"/NoDesktopShortcut /NoTaskbarShortcut"'`
+For example: `--params '"/NoAutostart /NoDesktopShortcut /NoTaskbarShortcut"'`
```

Comment:
It's not necessary to update this file at all. The description will be pulled from the README file at update.